### PR TITLE
WRQ-7108: Remove unnecessary vendor prefixes

### PR DESCRIPTION
**Pull request by vJIYEv**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Enact has CSS vendor prefixes. Most of them are not required anymore.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Considering browser versions we support, I removed unnecessary vendor prefixes.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-7108

### Comments
Enact-DCO-1.0-Signed-off-by: Jiye Kim (jiye.kim@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
@vJIYEv , Please update your branch with the latest develop.

File: packages/ui/styles/core.less [hunk 1]
```diff
@@ -39,17 +39,11 @@
 	// Selection
 	.enact-unselectable {
 		cursor: default;
-		-ms-user-select: none;
-		-webkit-user-select: none;
```
Comment (MikyungKim) [inline packages/ui/styles/core.less:43]: According to https://caniuse.com/?search=user-select, we need `-webkit-` prefix to use in Safari.

Comment (seunghoh) [reply packages/ui/styles/core.less:43]: @MikyungKim hawkeye~

Comment (vJIYEv) [reply packages/ui/styles/core.less:43]: OK

File: packages/ui/Scrollable/Scrollable.module.less [hunk 2]
```diff
@@ -35,8 +35,4 @@
 		overflow: hidden;
 		flex: 1 1 100%;
 	}
-
-	.contentNative {
```
Comment (MikyungKim) [inline packages/ui/Scrollable/Scrollable.module.less:39]: We need to remove the usage of this class from js also.

File: packages/ui/useScroll/useScroll.module.less [hunk 3]
```diff
@@ -37,8 +37,4 @@
 		overflow: hidden;
 		flex: 1 1 100%;
 	}
-
-	.scrollContentWrapperNative {
```
Comment (MikyungKim) [inline packages/ui/useScroll/useScroll.module.less:41]: Let's remove this from the code too.
https://github.com/enactjs/enact/blob/master/packages/ui/useScroll/useScroll.js#L1521

File: packages/ui/Scrollable/Scrollable.module.less (hunk 2 above)
Comment (vJIYEv) [reply packages/ui/Scrollable/Scrollable.module.less:39]: OK

File: packages/ui/useScroll/useScroll.module.less (hunk 3 above)
Comment (vJIYEv) [reply packages/ui/useScroll/useScroll.module.less:41]: OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]

**Pull request merged**